optimum/exporters/onnx/convert.py

```diff
@@ -46,6 +46,14 @@
 logger = logging.get_logger(__name__)  # pylint: disable=invalid-name
 
 
+class NumberOfInputsMatchError(ValueError):
+    pass
+
+
+class NumberOfOutputsMatchError(ValueError):
+    pass
```

Comment:
These are actually not used in `__main__.py` of the ONNX export, is it normal?

Reply:
Yes, will be used in a PR coming this week.